browser_patches/firefox/build.sh

```diff
@@ -41,6 +44,12 @@ echo "mk_add_options MOZ_OBJDIR=@TOPSRCDIR@/${OBJ_FOLDER}" >> .mozconfig
 if [[ $1 == "--juggler" ]]; then
   ./mach build faster
 else
+  # We manage Rust version ourselves.
```

Comment:
Should we cache the result to reduce impact on the build time?

Reply:
It should be quite instant, at least on Linux and Mac.
I'm yet to check no windows